src/d3a/models/market/one_sided.py

```diff
@@ -168,7 +171,9 @@ def _calculate_original_prices(cls, offer):
 
     def split_offer(self, original_offer, energy, orig_offer_price):
 
+        time_slot = original_offer.time_slot
```

Comment:
Nit, no need to declare a variable here, we can use `original_offer.time_slot` while passing the time_slot below

The same applies to the other assignment in the TwoSided market